ruby/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(age, parent_permission, name)
```

Comment:
- [ ] Please edit the `parent_permission` parameter to be true as default like so: `parent_permission = true`